yb-voyager/cmd/exportData.go

```diff
@@ -211,6 +211,13 @@ func exportData() bool {
 					utils.ErrExit("failed to delete stream id after data export: %v", err)
 				}
 			}
+			if exporterRole == SOURCE_DB_EXPORTER_ROLE {
+				msr, err := metaDB.GetMigrationStatusRecord()
+				if err != nil {
+					utils.ErrExit("get migration status record: %v", err)
+				}
+				deletePGReplicationSlot(msr, &source)
```

Comment:
What about the `PUBLICATION` objects?

Reply:
will take care of it in a separate PR once @makalaaneesh's corresponding changes are in.

Reply:
File a ticket for the pending work.